README.md

```diff
@@ -23,6 +23,9 @@ The following parameters require updates:
  - `discovery.apikey` - An API Key can be obtained from the api-manager for the user who has access to post the API.
 An API key can be created by logging into the APIC Manager UI and selecting the "My API Keys" option under the profile icon from the top navigation bar. 
 The apikey will be added to a kubernetes secret as part of the deployment and then mounted on the collector deployment pod.  
+ - `discovery.processor` - The flag to enable or disable the processors. The [configmap](apidiscovery/templates/processor-configmap.yaml) enables the existing batch and memory_limiter processor when `enabled` value is given and will not provide any processors in other cases. The processors should be enabled if the open telemetry collector needs to send a trace with more than 11000 spans. The available batch and memory limiter processors are configurable according to the requirements. <br /> &nbsp;
```

Comment:
...value is set and will...
....to send a trace with more than 11000 spans. If you attempt to send more than 11000 at a time the collector will reject the payload and you will be required to turn on the batch processor. <Example log of error>
....according to the base systems requirements.